### ISSUE TITLE
is a recursive blazon possible?

### ISSUE THREAD

**Issue by CastixGitHub** [karlwilcox/Blazon-Parser#3]
something like this:
http://www.phyast.pitt.edu/~micheles/scheme/_images/scheme_shield.png

**Comment by karlwilcox** [comment]
No, sorry. The aim of the parser is to support blazonry as practised in Mediaeval Europe, which pre-dates Church’s work on Lambda Calculus by several hundred years(!)

Having said that, if you wanted to blazon it you could say something like “in chief the identity ‘(Y F) = (F (Y F))’ the symbol lambda beside the shield repeated” and a heraldic artist would be able to draw it.
